tests/generators/OperationGenerator.ts

```diff
@@ -241,7 +222,6 @@ export default class OperationGenerator {
   public static async generateCreateOperationRequest (
     recoveryPublicKey: JwkEs256k,
     signingPublicKey: PublicKeyModel,
```

Comment:
To your point about non-operation keys should be separate from operation keys, I think we should add `updatePublicKey` between line 223 and 224 to disambiguate.

Reply:
Good catch.